integration_tests/conftest.py

```diff
@@ -96,7 +96,7 @@ def ingest_configs(datacube_env_name):
     }
 
 
-@pytest.fixture(params=["US/Pacific", "UTC", ])
+@pytest.fixture(params=["UTC", ])
```

Comment:
was this change intentional?

Reply:
Reverted the changes. Was pushed unintentionally.